src/main/java/org/jitsi/videobridge/VideoChannel.java

```diff
@@ -867,4 +867,23 @@ public void run()
             }
         };
     }
+
+    /**
+     * @return this {@link VideoChannel}'s {@link EndpointBase} as
+     * an {@link Endpoint} instance.
+     */
+    @Override
+    public Endpoint getEndpoint()
+    {
+        EndpointBase endpointBase = super.getEndpoint();
+        if (endpointBase instanceof Endpoint)
+        {
+            return (Endpoint) endpointBase;
+        }
+        else
+        {
+            throw new IllegalStateException(
```

Comment:
should this really throw here?  it seems like we have to check `instanceof` in a lot of places to check for `Endpoint` which defeats some of the advantages of the polymorphism there...could the octo/remote endpoints just have no-ops for the things we don't want them to do?

Reply:
I added getSelectedEndpoints() to EndpointBase (which allowed me to remove this method from VideoChannel), but I think get/set SctpConnection() should stay in Endpoint only.